bach/bach/series/series_json.py

```diff
@@ -524,6 +541,47 @@ def get_array_length(self) -> 'SeriesInt64':
             .copy_override_type(SeriesInt64) \
             .copy_override(expression=expression)
 
+    def array_contains(self, item: Union['Series', AllSupportedLiteralTypes]) -> 'SeriesBoolean':
+        """ For documentation, see implementation in parent class :class:`JsonAccessor` """
```

Comment:
```suggestion
        """ For documentation, see implementation in class :class:`JsonAccessor` """
```
I realize this is a copy-paste from the wrong docstring that I typed above, still good to have it correct